README.md

```diff
@@ -1,34 +1,34 @@
-# LaunchDarkly Sample Salesforce Apex Server-Side Application
+# LaunchDarkly sample Apex application
 
 We've built a simple application that demonstrates how LaunchDarkly's SDK works.
 
 Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit the [Apex SDK reference guide](https://docs.launchdarkly.com/sdk/server-side/apex).
 
-This guide requires the [Salesforce CLI](https://developer.salesforce.com/tools/sfdxcli), and the [Go compiler](https://golang.org/) to be installed.
+This guide requires you to install the [Salesforce CLI](https://developer.salesforce.com/tools/sfdxcli) and the [Go compiler](https://golang.org/).
 
-## Setup instructions
+## Build instructions
 
-1. Clone the Apex SDK
+1. Clone the Apex SDK.
 
 ```bash
 git clone https://github.com/launchdarkly/apex-server-sdk.git
 ```
 
-2. Deploy the SDK to Salesforce
+2. Deploy the SDK to Salesforce.
 
 ```bash
 cd apex-server-sdk
 sfdx force:source:deploy --targetusername='YOUR TARGET ORG' --sourcepath='force-app'
 ```
 
-3. Build the Salesforce LaunchDarkly bridge
+3. Build the Salesforce LaunchDarkly bridge.
 
 ```bash
 cd apex-server-sdk/bridge
 go build .
 ```
 
-4. Start the Salesforce bridge
+4. Start the Salesforce bridge. Set LD_SDK_KEY to your LaunchDarkly SDK key.
```

Comment:
```suggestion
4. Set the environment variable `LD_SDK_KEY` to your LaunchDarkly SDK key. Set environment variables for your Salesforce account.
5. Start the Salesforce bridge.
```
In the code sample, starting the bridge is at the end, so recommend calling out setting the env vars first.